saleor/graphql/checkout/mutations/checkout_shipping_address_update.py

```diff
@@ -163,7 +162,6 @@ def perform_mutation(
                 manager,
                 shipping_channel_listings,
             )
-        recalculate_checkout_discount(manager, checkout_info, lines, discounts)
```

Comment:
IMO we should recalculate discounts. When someone changes the shipping address, then the shipping method could change. Afterr the shipping method changed voucher could be inapplicable.

Reply:
This happens inside `fetch_checkout_prices_if_expired`